Geometry/DTGeometryBuilder/src/DTGeometryParsFromDD.cc

```diff
@@ -44,8 +45,8 @@ void DTGeometryParsFromDD::build(const DDCompactView* cview,
                                  RecoIdealGeometry& rig) {
 #ifdef EDM_ML_DEBUG
   edm::LogVerbatim("DTGeometry") << "DTGeometryParsFromDD::build";
-  static const string t0 = "DTGeometryParsFromDD::build";
-  TimeMe timer(t0, true);
+  static const std::string t0 = "DTGeometryParsFromDD::build";
```

Comment:
This should also get commented out.
@bsunanda please have a look at the alternative implementation proposed in https://github.com/cms-sw/cmssw/pull/41243. That one looks cleaner to me: if the `TimeMe` does not exist here, why to let it in the code, even if commented out?